crates/connector_proxy/src/connector_runner.rs

```diff
@@ -13,8 +15,19 @@ use proto_flow::capture::PullResponse;
 use proto_flow::flow::DriverCheckpoint;
 use tokio::io::copy;
 use tokio::process::{ChildStdin, ChildStdout};
+use tokio::task::JoinHandle;
 use tokio_util::io::{ReaderStream, StreamReader};
 
+async fn flatten_join_handle<T, E: std::convert::From<tokio::task::JoinError>>(
```

Comment:
I think there's a borked merge with upstream -- I'm seeing a bunch of recent changes now being unintentionally reverted in this PR.

Reply:
Oh gross...